Documentation/Blazorise.Docs/Pages/Docs/Components/Checks/CheckPage.razor

```diff
@@ -24,6 +24,16 @@
     <DocsPageSectionSource Code="CheckExample" />
 </DocsPageSection>
 
+<DocsPageSection>
+    <DocsPageSectionHeader Title="Check with name">
+        Simple check group
+    </DocsPageSectionHeader>
+    <DocsPageSectionContent Outlined>
+        <CheckExampleWithName />
+    </DocsPageSectionContent>
+    <DocsPageSectionSource Code="CheckExampleWithName" />
```

Comment:
The example filename must end with Example, e.g., `CheckWithNameExample.razor`, and must be placed in the `/Examples` subfolder.

After that you just run the `Blazorise.Docs.Compiler` with CTRL+f5 to generate what need to be generated.